Frontend/CO.CDP.OrganisationApp/Pages/Forms/_FormElementMultiLineInput.cshtml

```diff
@@ -2,36 +2,91 @@
 @using CO.CDP.Localization
 
 @{
-    var hasError = Model.HasValidationError(nameof(Model.TextInput), ViewData.ModelState);
-}
-
-<div class="govuk-form-group @(hasError ? "govuk-form-group--error" : "") govuk-character-count" data-module="govuk-character-count" data-maxlength="10000" data-threshold="90">
+    var textInputHasError = Model.HasValidationError(nameof(Model.TextInput), ViewData.ModelState);
+    var selectOptionHasError = Model.HasValidationError(nameof(Model.HasValue), ViewData.ModelState);
+    string textInputAriaDescribedby = "";
```

Comment:
I know this is an existing pattern but it could be improved by using a list or separate method to avoid string concatenation